config.toml

```diff
@@ -9,6 +9,12 @@
 # Default:
 #   bind_port = 8080
 
+# Path to directory with static control API specs.
+#
+# Default:
+#   static_specs_path = None
+static_specs_path = "dev/specs"
```

Comment:
It should not be `None`. It should have a default value like `specs/`. And be omitted if no such dir exists (and emit `WARN` log message in such situations).